### PR TITLE
Name the team better

### PR DESCRIPTION
**Pull request by paulo-ferraz-oliveira**
(Brujo is sad that our current name is so humble)

Just throwing this out there.

🔥 and shuffle us around to (was sorted alphabetically, but "randomness" is better)

### CODE REVIEW

**Review by elbrujohalcon — APPROVED** [review]

**Review by pablocostass — APPROVED** [review]

**Review by filipevarjao — APPROVED** [review]

File: _config.yml [hunk 1]
```diff
@@ -66,7 +66,7 @@ languages:
     image: img/languages/otpcl.svg
 
 organizer:
-  name: Some kind folks from the BEAM community
+  name: The Fellowship of the BEAM
```
Comment (starbelly) [inline _config.yml:69]: Perfect! One BEAM to rule them all!

**Comment by starbelly** [comment]
@paulo-ferraz-oliveira Suggestion, have Fellowship of the BEAM link to the organizers section in the footer.

File: _config.yml (hunk 1 above)
Comment (paulo-ferraz-oliveira) [reply _config.yml:69]: And in the VM bind them...

**Comment by paulo-ferraz-oliveira** [comment]
> @paulo-ferraz-oliveira Suggestion, have Fellowship of the BEAM link to the organizers section in the footer.

That should be done 😕 I added a `#` there.

**Comment by paulo-ferraz-oliveira** [comment]
Hoping Bryan's comment was the only reason I didn't get his approval, I'm squashing, merging and deploying (I'm actually not deploying, GitHub is...).

**Pull request merged**